.github/workflows/release.yml

```diff
@@ -93,7 +93,7 @@ jobs:
     runs-on: [self-hosted, public, linux, x64]
     environment: release
     env:
-      PYTHON_VERSION: "3.8"
+      PYTHON_VERSION: "3.7"
```

Comment:
@gruebel Why?

Reply:
because our base version in `checkov` is 3.7 and the last time, when the CI tried to do it with 3.8 it failed.